src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```diff
@@ -14,26 +14,7 @@
 import java.util.stream.Collectors;
 
 import org.apache.commons.lang.Validate;
-import org.bukkit.BlockChangeDelegate;
-import org.bukkit.Bukkit;
-import org.bukkit.Chunk;
-import org.bukkit.ChunkSnapshot;
-import org.bukkit.Difficulty;
-import org.bukkit.Effect;
-import org.bukkit.FluidCollisionMode;
-import org.bukkit.GameRule;
-import org.bukkit.HeightMap;
-import org.bukkit.Location;
-import org.bukkit.Material;
-import org.bukkit.Particle;
-import org.bukkit.Raid;
-import org.bukkit.Sound;
-import org.bukkit.SoundCategory;
-import org.bukkit.StructureType;
-import org.bukkit.TreeType;
-import org.bukkit.World;
-import org.bukkit.WorldBorder;
-import org.bukkit.WorldType;
+import org.bukkit.*;
```

Comment:
Please avoid wildcard imports

Reply:
Not quite sure why you dont like wildcard imports, they look way nicer, but whatever. Took me ages to find out how to make my ide disable them. Your code formatting conventions are annoying af

Reply:
> they look way nicer, but whatever

Wildcard imports are not a style choice, they are a functional utility.
Their purpose is to import entire namespaces which can reduce the file size when you got many imports from the same namespace. However, the 90s are over, so we don't need to worry about every single bit or byte in our file anymore.

Nowadays, wildcard imports are sometimes regarded as a bad practice as they lead to so-called namespace pollution.
By nature, wildcard imports will automatically import any classes from the given namespace into the current class.
Overuse of this can lead to conflicts and collisions ([Wikipedia](https://en.wikipedia.org/wiki/Name_collision)), as the whole point of a namespace is to seperate class names which are non-unique.
By using a wildcard import, you are opening the door to ambiguities and potential collisions.

Check this for more info and a pretty good example:
https://rules.sonarsource.com/java/RSPEC-2208

And as you can guess, the `org.bukkit` namespace is by no means a small one...

Reply:
99% sure it gets optimized on compilation, but whatever

Reply:
This is not a runtime issue, it's a bad development practice and not happening on the compiler level.
But either way, I gave you a decent explanation, what you do with this information and advice is all your call.